docs/snapshot-single-delete/scenarios/_query_tree.py

```
@@ -0,0 +1,93 @@
+    return out, err
+
+def mysql(client, sql):
+    cmd = "mysql -pzstack.mysql.password zstack -t -e \"" + sql.replace('"', '\\"') + "\""
+    out, err = run(client, cmd)
+    # Filter out mysql password warning
+    err = "\n".join([l for l in err.splitlines() if "Using a password" not in l and l.strip()])
+    return out, err
+
+def main():
+    client = paramiko.SSHClient()
+    client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
```

Ruff (0.15.12)

[error] Paramiko call with policy set to automatically trust the unknown host key

(S507)